components/land-use/LandUseTable.vue

```diff
@@ -1,15 +1,32 @@
 <template>
-    <v-container fluid class="overflow-auto container-height">
-        <v-toolbar absolute min-width="100vw" color="secondary">
-            <v-btn icon>
-                <v-icon color="#FFFFFF">mdi-table</v-icon>
-            </v-btn>
-        </v-toolbar>
-        <v-card class="py-8 mt-8">
-            <v-card-title>
-                <v-row>
+    <v-row>
+        <v-dialog
+            v-model="tableLandUse"
+            transition="dialog-bottom-transition"
+            persistent
+            no-click-animation
+            hide-overlay
+            width="75vw"
+        >
+            <v-card>
+                <v-toolbar class="background__toolbar" dark color="primary">
+                    <h3>{{ $t('table-name') }}</h3>
                     <v-spacer></v-spacer>
-                    <a>
+                    <v-btn icon @click="checkUpdate()">
+                        <v-icon>mdi-close</v-icon>
+                    </v-btn>
+                </v-toolbar>
+            </v-card>
+            <v-container fluid white>
+                <v-skeleton-loader
+                    v-if="isLoadingTable"
+                    type="table-row-divider@8"
+                ></v-skeleton-loader>
+                <v-card v-if="isLoadingTable === false">
```

Comment:
`!!!isLoadingTable`

Reply:
Sorry, forgot to delete these components

Resolved.